src/components/Landing/index.tsx

```diff
@@ -111,7 +111,7 @@ export default function Landing() {
       ].map((data, i) => (
         <Space
           style={{ fontStyle: 'italic', paddingLeft: 8 }}
-          key={i}
+          key={`builtfor_${i}`}
```

Comment:
I think you're better off just using `i` here. I don't think the extra text will make a diff to performance